ocaml/xapi/xapi_db_upgrade.ml

```diff
@@ -867,7 +867,8 @@ let update_livepatch_guidance =
        reboot_host_on_xen_livepatch_failure in \
        host.pending_guidances_recommended. 2. Move the rest guidances in \
        host.pending_guidances into host.pending_guidances_recommended"
-  ; version= (fun _ -> true)
+      (* TODO: update below schema version to which the feature branch got merged with *)
+  ; version= (fun x -> x < (5, 769))
```

Comment:
I recommend adding a failure to the CI so this note is not forgotten when merging to master:
```
let update_livepatch_guidance =
  let change_version_to_the_latest_schema_version = () in
  {
    description=
      "1. Replace reboot_host_on_livepatch_failure ....
```

This will mean that there might be some errors creeping before merging to master, but those can be fixed at that time and they tend to be rare